frontend/src/pages/org/workflow-editor.ts

```diff
@@ -263,6 +263,9 @@ export class CrawlConfigEditor extends LiteElement {
   @property({ type: Boolean })
   orgExecutionMinutesQuotaReached = false;
 
+  @property({ type: Boolean })
+  newWorkflow = false;
```

Comment:
You can use the `configId` (workflow ID) property to check if this is a new workflow!

Reply:
oh good call, I missed that!